ndk-build/src/apk.rs

```diff
@@ -155,6 +160,41 @@ impl<'a> UnalignedApk<'a> {
         Ok(())
     }
 
+    pub fn add_file(&self, base_path: &Path, path: &Path) -> Result<(), NdkError> {
+        let out = self.0.build_dir.join(path);
+        let out_dir = out.parent().unwrap();
+        std::fs::create_dir_all(&out_dir)?;
+        std::fs::copy(base_path.join(path), out)?;
+
+        let mut aapt = self.0.build_tool(bin!("aapt"))?;
+        aapt.arg("add").arg(self.0.unaligned_apk()).arg(path);
```

Comment:
Should we make sure that `aapt` runs relative to the build directory here? There are environment variables dealing with moved `target/` directories and I don't exactly remember how they are handled here, if we keep running in the right PWD.

If we do that this might as well run `aapt` with `base_path` as `PWD`.